hack/lib/util.sh

```diff
@@ -67,15 +65,25 @@ kube::util::wait_for_url() {
   return 1
 }
 
-kube::util::wait_for_url_with_bearer_token() {
+kube::util::wait_for_url() {
   local url=$1
-  local token=$2
+  local prefix=${2:-}
+  local wait=${3:-1}
+  local times=${4:-30}
+  local maxtime=${5:-1}
+
+  kube::util::curl "${url}" "${prefix}" "${wait}" "${times}" "${maxtime}"
+}
+
+kube::util::wait_for_url_with_bearer_token() {
+  local url=${1}
+  local token=${2}
   local prefix=${3:-}
   local wait=${4:-1}
   local times=${5:-30}
   local maxtime=${6:-1}
 
-  kube::util::wait_for_url "${url}" "${prefix}" "${wait}" "${times}" "${maxtime}" "--oauth2-bearer ${token}"
```

Comment:
could this PR be simplified to just change this line to `-H \"Authorization: Bearer ${token}\"`?

Reply:
I have tried this before, but due to the word splitting mechanism in bash, curl will see `-H` `"Authorization:` `Bearer` `${token}"`, which is not expected.

Reply:
what about using `"${@:6}"` instead of `extra_args`?

Reply:
(I'm trying to avoid a third utility method, and minimize change, if possible)

Reply:
sound like a good idea, I just learn that we could do slicing in bash